export_snowflake/db_sync.py

```diff
@@ -510,23 +505,74 @@ def load_file(self, s3_key, count, size_bytes):
         self.logger.info(
             'Loading into %s: %s',
             self.table_name(stream, False),
-            json.dumps({'inserts': inserts, 'updates': updates, 'size_bytes': size_bytes})
+            json.dumps({'inserts': inserts, 'updates': updates})
         )
 
-    def _load_file_merge(self, s3_key, stream, columns_with_trans) -> Tuple[int, int]:
+    def use_default_schema(self):
+        return f"USE SCHEMA {self.schema_name}"
+    
+    def get_temporary_stage_name(self):
+        """Generate snowflake stage name"""
+        stream = self.connection_config.get('stream', None)
+        if stream:
+            return f"export_{stream}_temp_stage"
```

Comment:
should we use random strings instead? If the stage exists already, wouldn't our code overwrite existing stage?

Reply:
update: use export_task_id instead of stream name in case we export to the same table
exportTaskID is from base64url, replace the potential - with _